packages/frontend/src/components/order-table/OrderTable.jsx

```diff
@@ -23,13 +23,15 @@ export default function OrderTable({ rows }) {
           {rows.map((row) => (
             <TableRow key={row.id}>
               <TableCell align="center">{row.id}</TableCell>
-              <TableCell align="center">{row.phone}</TableCell>
-              <TableCell align="center">{row.street}</TableCell>
+              <TableCell align="center">{row.mainPhone}</TableCell>
+              <TableCell align="center">{row.address.streetName}</TableCell>
             </TableRow>
           ))}
         </TableBody>
       </Table>
     </Paper>
+  ) : (
+    <p className="noOrders">Таких заказов нет, проверь критерии поиска</p>
```

Comment:
css классы обычно называют в kebab-case (через дефис), давай и этот так назовем:
```suggestion
    <p className="no-orders">Таких заказов нет, проверь критерии поиска</p>
```